nilearn/decoding/decoder.py

```diff
@@ -1497,11 +1512,9 @@ def fit(self, X, y, groups=None):
 
         %(groups)s
 
-        %(base_decoder_fit_attributes)s
-
         """
         check_params(self.__dict__)
-        self.classes_ = ["beta"]
+        self._classes_ = ["beta"]
```

Comment:
regressor cannot have classes (at least not as public attributes that will be documented)